server/application-version.js

```diff
@@ -0,0 +1,8 @@
+// eslint-disable import/no-unresolved,global-require
```

Comment:
You should take out the `eslint` comments as we don't use it.

Reply:
Yep sure I'll take out now